pkg/pillar/zedcloud/send.go

```diff
@@ -171,11 +184,13 @@ func VerifyAllIntf(ctx *ZedCloudContext,
 				log.Errorf("Zedcloud un-reachable via interface %s: %s",
 					intf, err)
 				errorList = append(errorList, err)
+				intfErrMap.AddError(intf, err, errSrc)
 				continue
 			}
 			switch resp.StatusCode {
 			case http.StatusOK, http.StatusCreated:
 				log.Debugf("VerifyAllIntf: Zedcloud reachable via interface %s", intf)
+				intfErrMap.DelError(intf)
```

Comment:
In which case does this DelError delete an error? We are trying each interface once. Hence there wouldn't be any existing error for the interface. And having complexity based on an incorrect understanding of the code just makes complexity explode because the next reader thinks this needs to handle these cases that does not exist.

Reply:
Would "ClearIntfErrors()" be a better name?
 Here, we are retrying the test.. If we see errors the first time on an interface - but the next time it might succeed. So any errors in the intfErrMap for that interface need to be cleared.